docs/components/emails.rst

```diff
@@ -1,13 +1,434 @@
 Emails
```

Comment:
This one's big. It removes outdated things that have been removed in Mautic 3 or 4 (like `addTokenSection`). It also adds some new things, like docs on Email Transport/Email providers.

Original docs: https://developer.mautic.org/#extending-emails